### ISSUE TITLE
Currency units stop line from producing an output

### ISSUE THREAD

**Issue by Ziraya** [nortakales/vs-code-qalc#32]
If I put a currency unit, USD in my case, in a line anywhere, in any form, that line no longer has an output.

```qalc
5                               = 5
$5
5 $
$5 in EUR
5USD
5 USD
5 usd
5usd
5 USD
five = 5                        = 5
five USD
five km                         = 5 km
```

I have tried disabling `qalc.currency.convertLocal`, no effect

**Comment by nortakales** [comment]
The API I was using for exchange rates has just revoked free access. I can push a small change very quickly to at least restore `$` and `USD`. I will need to find a new API that gets us as much free access as possible for the extension to restore actual conversions.

**Comment by nortakales** [comment]
I'm consistently getting around 14-15 seconds, which admittedly is pretty bad. I need to look into this. I'll open a new issue to track it.

**Comment by nortakales** [comment]
Currency and exchange rates should be fixed in `0.1.14`
